ui.apps/karma.conf.js

```diff
@@ -30,7 +30,6 @@ module.exports = function(config) {
     files: [
       'node_modules/@babel/polyfill/dist/polyfill.js',
       'src/main/content/jcr_root/apps/core/cif/components/commerce/product/**/js/*.js',
-      'src/main/content/jcr_root/apps/core/cif/components/commerce/productlist/**/js/*.js',
```

Comment:
I think was messed up a bit in #263. Not sure why this works or not works? 

For the productcollection component we have tests at https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/test/components/commerce/productcollection/productCollectionTest.js. Those tests require the clientlib at https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v1/productcollection/clientlibs/js/productcollection.js.

Reply:
The file `productcollection.js` is directly imported in `productCollectionTest.js` so that's why it works. If we want to clean this up, we should create another issue as this is not related to my change (I removed that line because there was a warning in the build that there isn't any matching file with that path).